packages/manager/modules/vrack/src/select-dialog/template.html

```diff
@@ -11,7 +11,7 @@
                     class="ng-isolate-scope oui-modal oui-modal_shadow oui-modal_warning"
                 >
                     <p
-                        data-translate="vrack_select_dialog_modal_description"
+                        data-ng-bind="($ctrl.isIpv6 ? 'vrack_select_dialog_modal_description_for_ipv6': 'vrack_select_dialog_modal_description_for_vs') | translate"
```

Comment:
I think we should use a variable for vrack services as well (isVrackServices)

Reply:
Yes I taught of same design. In that case I need to add another <p> tag with the content for each case.